spring-cloud-gateway-integration-tests/grpc/src/main/java/org/springframework/cloud/gateway/tests/grpc/GRPCApplication.java

```diff
@@ -18,31 +18,27 @@
 
 import java.io.File;
 import java.io.IOException;
-import java.security.cert.X509Certificate;
 import java.util.concurrent.TimeUnit;
 
 import javax.net.ssl.SSLException;
-import javax.net.ssl.TrustManager;
-import javax.net.ssl.X509TrustManager;
 
 import io.grpc.Grpc;
 import io.grpc.Server;
 import io.grpc.ServerCredentials;
 import io.grpc.TlsServerCredentials;
+import io.grpc.netty.shaded.io.grpc.netty.NettySslContextServerCredentials;
+import io.grpc.netty.shaded.io.netty.handler.ssl.ApplicationProtocolConfig;
+import io.grpc.netty.shaded.io.netty.handler.ssl.ApplicationProtocolNames;
+import io.grpc.netty.shaded.io.netty.handler.ssl.SslContextBuilder;
```

Comment:
Using shaded netty seems off to me.

Reply:
Luckily it was used in an old method, I could remove it without changing the code. Thanks for the keen eyes!

Reply:
I found and fixes all usages of "shadded" pacakges. It was indeed and issue that needed solving, thanks!